src/main/kotlin/anttracker/db/SetupSchema.kt

```diff
@@ -122,10 +141,25 @@ class Issue(
     var product by ProductEntity referencedOn Issues.product
     var anticipatedRelease by Release referencedOn Issues.anticipatedRelease
     var creationDate by Issues.creationDate
-    var status by Issues.status
+    private var _status by Issues.status
+    var status: Status
+        set(newStatus) {
+            _status = newStatus.toString()
+        }
+        get() = requireNotNull(_status.toStatus())
```

Comment:
Is there a case where _status would return null?

Reply:
`toStatus` should never return null, although the return type allows for it. The assertion is there as a sanity check.